examples/bedrock-claude-factual-knowledge.ipynb

```diff
@@ -11,9 +11,9 @@
     "tags": []
```

Comment:
<p>I think that almost all users (aside from contributors who are very familiar with the internals of our codebase) no longer need to be aware of this env variable. We originally needed to include this as a hacky way to avoid encountering some niche Ray bugs, but I have since fixed those bugs in this <a href="https://github.com/aws/fmeval/pull/115" target="_blank">PR</a>. I think it's best not to expose details about infra internals (i.e. Ray details) as much as possible, so as to not overwhelm the average user.</p><p> </p><p>This env variable is currently only being used to determine the number of actors in the <code>ActorPools</code> that we create. While too many actors can lead to OOM issues <em>in theory</em>, in our use case, OOM issues are almost never caused by having too many <code>ActorPool</code>  actors, since those actors are always cheap (memory and CPU usage-wise).</p><p> </p><p>The only time OOM would be an issue for us in the example notebooks is if we're dealing with an eval algo that uses the <code>BertscoreHelperModel</code>, which is a relatively expensive (memory-wise) actor. But in this cases, we don't use <code>ActorPool</code>, so <code>PARALLELIZATION_FACTOR</code> doesn't apply. In such cases, the user just needs to use a notebook instance that has enough memory to load the Bert model.</p>